src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1460,7 +1461,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return s_variablesMappings.GetValue(
                     Me,
                     Function(BoundUsing)
-                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration))
+                        Dim usingStatementSyntax = Syntax.ChildNodes().OfType(Of UsingStatementSyntax).SingleOrDefault()
+                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration), usingStatementSyntax)
```

Comment:
>Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration), usingStatementSyntax) [](start = 24, length = 95)

We are fixing null syntax, but it looks like IVariableDeclarationStatement.Declarations can return default array for this object. Are we fine leaving product in this state? It seems addressing this issue is as easy as checking if BoundUsing.ResourceList is default.